pkg/gardenlet/operation/botanist/operatingsystemconfig.go

```diff
@@ -93,8 +93,13 @@ func (b *Botanist) DeployOperatingSystemConfig(ctx context.Context) error {
 		return fmt.Errorf("secret %q not found", v1beta1constants.SecretNameCACluster)
 	}
 
```

Comment:
IMO, we could combine them as they are related logically and the final goal of this block is to fetch the `clusterCABundle`.

```suggestion
```